indico/modules/events/abstracts/templates/management/abstract.html

```diff
@@ -1,6 +1,143 @@
 {% extends 'events/abstracts/management/_base.html' %}
-{% from 'events/abstracts/_abstract.html' import render_abstract_public %}
+{% from 'events/abstracts/_abstract.html' import render_abstract_public, render_abstract_comments, render_edited_hint %}
 
 {% block content %}
     {{ render_abstract_public(abstract, management=true) }}
+    <div id="abstract-private" class="i-timeline">
+        <div class="i-timetable-connect-down"></div>
+        <div class="i-timeline-separator"></div>
+        {{ render_abstract_comments(abstract) }}
+        {{ _render_abstract_reviews(abstract) }}
+        <div class="i-timeline-separator"></div>
+        <div class="i-timetable-connect-up"></div>
+        {{ _render_decision_box(abstract) }}
+    </div>
 {% endblock %}
+
+{% macro _render_abstract_reviews(abstract) %}
+    {% for track in abstract.proposed_for_tracks %}
+        {% set reviews = track.abstract_reviews|selectattr('abstract', 'equalto', abstract)|list %}
```

Comment:
Wouldn't it be cleaner to get the reviews of the abstract and filter by track? There are most likely less reviews on an abstract than there are for a whole track.